SofaKernel/modules/SofaCore/src/sofa/core/objectmodel/Data.h

```diff
@@ -239,10 +239,10 @@ class Data : public BaseData
         return out;
     }
 
-    SOFA_DEPRECATE_DATA_API("Deprecated before definitive removal. Please update your code by replacing 'myData == aValue' with 'myData.getValue() == aValue'")
+    SOFA_ATTRIBUTE_DISABLED("2020-12-01 PR#1639", "2021-02-08 PR#1793", "Update your code by replacing 'myData == aValue' with 'myData.getValue() == aValue'")
```

Comment:
Again a location where we should use a two level macro
eg: 
SOFA_DEPRECATE_DATA_API_DISABLE_EQUALITY_OPERATOR()